build.gradle

```diff
@@ -37,35 +29,34 @@ dependencies {
 
   //Gradle doesn't yet handle dependency resolution conflicts for localGroovy()
   //See: http://forums.gradle.org/gradle/topics/conflicting_module_versions_with_groovy_gradle_2_0_rc_1
-  testCompile('com.netflix.nebula:nebula-test:2.2.0') {
+  testCompile('com.netflix.nebula:nebula-test:2.2.1') {
     exclude group:'org.codehaus.groovy'
   }
 
   //Allow testing of optional plugin configuration
   testCompile (
     'org.jfrog.buildinfo:build-info-extractor-gradle:3.0.1',
-    'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.0',
+    'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2',
+    'com.gradle.publish:plugin-publish-plugin:0.9.0',
   )
 }
 
-test {
-  maxParallelForks = 2
+tasks.withType(Test) {
+  maxParallelForks = Runtime.runtime.availableProcessors()
 }
 
-plugindev {
-  authorEmail = 'stormbeta@gmail.com'
-  authorId = 'stormbeta'
-  authorName = 'Jason Miller'
-  pluginDescription "CI Lifecycle and Conventions"
-  pluginId 'com.readytalk.ci'
-  pluginImplementationClass 'com.readytalk.gradle.plugins.CiLifecyclePlugin'
-  pluginLicenses 'MIT'
-  pluginTags 'gradle', 'conventions', 'publishing', 'ci'
-  projectInceptionYear '2014'
-  projectIssuesUrl = 'https://github.com/ReadyTalk/gradle-readytalk-ci/issues'
-  projectUrl = 'http://oss.readytalk.com/gradle-readytalk-ci'
-  projectVcsUrl = 'https://github.com/ReadyTalk/gradle-readytalk-ci.git'
-  done()
+pluginBundle {
+  website = 'http://oss.readytalk.com/gradle-readytalk-ci'
+  vcsUrl = 'https://github.com/ReadyTalk/gradle-readytalk-ci.git'
+  description = "CI Lifecycle and Conventions"
+  tags = [ 'gradle', 'conventions', 'publishing', 'ci' ]
+
+  plugins {
+    ciPlugin {
+      id = 'com.readytalk.ci'
```

Comment:
Eventually I'd like the different pieces to be individually applied using the public ids, similar to how the nebula plugins are laid out.

For now I think it's fine to have just the top-level, since individual plugins can still be applied using the buildscript classpath method of including a plugin and because only integTest is actually independent.